contributors/yash-raj-bharti/index.md

```diff
@@ -0,0 +1,37 @@
+
+I still remember my first contribution to MDN. I was so proud of myself. Even though the fix was small, the cheer from Will Bamberg(a contributor) at the time meant a lot. It was for a [PWA](https://github.com/mdn/content/pull/29488), and I said: iTunes value associated with the App Store for platform members of related apps has been dropped since it's misleading and currently not supported by Safari. The list has been updated as per the W3C wiki.
+
+Since then, I have been lurking to find a way to contribute to MDN, I also contributed to web.dev to learn more. I am not new to MDN W3C, or Web.dev (HTML5Rocks), and have been learning like a nerd for the last 9 years, I was a couple of years back I started contributing, felt like someone helping the space, making explainers, worked alongside people like Adam Argyle from [CSS motion blur](https://github.com/w3c/csswg-drafts/issues/11134) and Thomas Steiner(Project Fugu). I discovered, that when I put my passion and heart into it, I can achieve, the grit to solve challenges isn't new to me.
+
+My work at Liquid Galaxy, documentation and creating an innovative UX to control Google Earth using Google Maps (a first ever!) led me to get invited as a Speaker to Google, CA, USA (my first time outside India), where I got to give a Talks at Google, and most importantly network! I met Kristi Progri who was previously a Mozilla Representative, and I felt that's so cool, I read from Mozilla every day and was confident too that I could do something.
+
+This is when after an incredible journey to Google, knowing the roots of the liquid galaxy from 2008, I was mesmerized and was full of energy to do something, thus I went to MDN and saw some simpler bugs like buttontext, where I met Estelle Weyl (and so thankful I did, as I had all the passion and pure knowledge waiting to test the waters). They helped me with the contributions by stating how buttontext for accessibility reasons will be fixed by filing it to Webkit. After some PRs like this, I mailed them asking to create an issue about the `<model>` tag, what result? They told me it's still a WIP but there are some Widely Baseline Available PRs I can contribute to. They "trusted me", created a Google Sheet, and gave me edit access. And I delivered. The rest is history, I found my chance and took it, I knew Estelle because I had read her book on CSS, and was amazed to work alongside her on this. [This sheet](https://docs.google.com/spreadsheets/d/1O9tOVsZDF26A8Rag38rAhGWRH5l6VTbDCJIknyKzO_0/edit?gid=276647839#gid=276647839) holds value to me and is the most updated thing on the work we are making.
+
+I love it because just think about how many future developers can be helped with it, those who truly learn like me and do not rely on crash courses to just get it "done", I thank Estelle for my journey and I want to stay in this community and help make mdn the library, the well-organized library I know It as, where you just know where each book is.
```

Comment:
```suggestion
I love it because just think about how many future developers can be helped with it, those who truly learn like me and do not rely on crash courses to just get it "done". I thank Estelle for my journey and I want to stay in this community and help make MDN the library, a well-organized library I know it as, where you just know where each book is.
```